public/ballers_new.sql

```diff
@@ -0,0 +1,165 @@
+-- phpMyAdmin SQL Dump
+-- version 5.0.1
+-- https://www.phpmyadmin.net/
+--
+-- Host: localhost
+-- Generation Time: Jun 04, 2020 at 09:13 PM
+-- Server version: 10.4.11-MariaDB
+-- PHP Version: 7.4.2
+
+SET SQL_MODE = "NO_AUTO_VALUE_ON_ZERO";
+SET AUTOCOMMIT = 0;
+START TRANSACTION;
+SET time_zone = "+00:00";
+
+
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
+/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
+/*!40101 SET NAMES utf8mb4 */;
+
+--
+-- Database: `ballers_new`
+--
+
+-- --------------------------------------------------------
+
+--
+-- Table structure for table `area`
+--
+
+CREATE TABLE `area` (
+  `area_id` int(255) NOT NULL,
+  `area_name` text NOT NULL,
+  `state_id` int(255) NOT NULL
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
+
+--
+-- Dumping data for table `area`
+--
+
+INSERT INTO `area` (`area_id`, `area_name`, `state_id`) VALUES
+(1, 'Lekki Phase 1', 1),
+(2, 'Orchid Hotel Road', 1),
+(3, 'Wuse II', 2);
+
+-- --------------------------------------------------------
+
+--
+-- Table structure for table `houses`
+--
+
+CREATE TABLE `houses` (
+  `house_id` int(255) NOT NULL,
+  `type` varchar(255) NOT NULL,
+  `price` varchar(255) NOT NULL,
+  `area_id` int(255) NOT NULL,
+  `state_id` int(255) NOT NULL,
+  `longitude` varchar(255) NOT NULL,
```

Comment:
Can we make the longitude and latitude of the houses to be optional by accepting NULL values, please? This will save us from issues later if the longitude and latitude of the house is not known when the house is saved in the database.